pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/util/position/ChunkPosition.kt

```diff
@@ -17,15 +17,23 @@ class ChunkPosition(world: World?, val x: Int, val z: Int) {
         get() = (x.toLong() shl 32) or (z.toLong() and 0xFFFFFFFFL)
 
     val chunk: Chunk?
-            get() = world?.getChunkAt(x, z)
+        get() = world?.getChunkAt(x, z)
+
+    /**
+     * Obtaining an instance of a Chunk (eg through block.getChunk()) will
+     * often LOAD THE CHUNK ITSELF. Yes, actually. This method does not
+     * load the chunk, so it's a safe way to check if a chunk is loaded.
+     */
+    val loaded: Boolean
```

Comment:
```suggestion
    val isLoaded: Boolean
```
Nitpick, feel free to ignore